crates/release_plz_core/src/command/release.rs

```diff
@@ -65,6 +67,7 @@ impl ReleaseRequest {
             packages_config: PackagesConfig::default(),
             publish_timeout: minutes_30,
             release_always: true,
+            branch_prefix: BRANCH_PREFIX.to_string(),
```

Comment:
can you rename `BRANCH_PREFIX` to `DEFAULT_BRANCH_PREFIX`?

Reply:
renamed `BRANCH_PREFIX` to `DEFAULT_BRANCH_PREFIX`